### DISCUSSION TITLE
Create Github pages website

### DISCUSSION THREAD

**Discussion by ePirat** [pupnp/pupnp#350] [Ideas]
As the project is now part of a Github Org, we could create a Github Pages based website for the project/org. This way people could easily contribute to the pupnp website, just like they can to libupnp using Github Pull requests.

It would become available at pupnp.github.io.

**Comment by mrjimenez** [comment]
We will have to change a few things on the docs folder, right? Or is it better to use a different branch?

**Comment by ePirat** [comment]
It would be in a different repo, not the pupnp repo, to have a website for the org using gitlab pages.
And for the docs probably a docs repository with subfolders for each version with the HTML and PDF docs, to not clutter the main pages repo and possibly even automate docs creation for releases without needing to grant access to the "full" website repo.

**Comment by ePirat** [comment]
@mrjimenez Any update on this? Or should I just prepare one and then you decide if we want to use it or not?

**Comment by mrjimenez** [comment]
Hi!

I really could not look into this issue.
If you prepare one, we can go from there.

Thanks!

**Comment by ePirat** [comment]
Ok, great! Sorry for the late reply, for whatever reason I did not get a mail for your reply so I missed it completely…

**Comment by mrjimenez** [comment]
No problem!

**Comment by mrjimenez** [comment]
Just for reference, we have this up: https://pupnp.github.io/pupnp/

**Comment by Vollstrecker** [comment]
Nice page, 1.14.9 (I thought this one was skipped) is the latest release.
